--- uvicorn-0.46.0/pyproject.toml
+++ uvicorn-0.46.0/pyproject.toml
@@ -1,14 +1,16 @@
 [build-system]
-requires = ["hatchling"]
-build-backend = "hatchling.build"
+requires = ["setuptools>=61"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools.packages.find]
+include = ["uvicorn*"]
 
 [project]
 name = "uvicorn"
-dynamic = ["version"]
+version = "0.46.0"
 description = "The lightning-fast ASGI server."
 readme = "README.md"
-license = "BSD-3-Clause"
-license-files = ["LICENSE.md"]
+license = { text = "BSD-3-Clause" }
 requires-python = ">=3.10"
 authors = [
     { name = "Tom Christie", email = "tom@tomchristie.com" },
@@ -93,8 +95,6 @@
 Homepage = "https://uvicorn.dev/"
 Source = "https://github.com/Kludex/uvicorn"
 
-[tool.hatch.version]
-path = "uvicorn/__init__.py"
 
 [tool.hatch.build.targets.sdist]
 include = ["/uvicorn", "/tests"]
