--- typer-0.25.1/pyproject.toml
+++ typer-0.25.1/pyproject.toml
@@ -1,15 +1,15 @@
 [build-system]
 requires = [
-    "pdm-backend",
+    "setuptools>=61",
 ]
-build-backend = "pdm.backend"
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools.packages.find]
+include = ["typer*"]
 
 [project]
 name = "typer"
-license = "MIT"
-license-files = [
-    "LICENSE",
-]
+license = { text = "MIT" }
 dynamic = []
 description = "Typer, build great CLIs. Easy to code. Based on Python type hints."
 authors = [
